src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/Example1.java

```diff
@@ -23,6 +23,10 @@
 
 import java.util.List;
 
+import java.util.function.Function;
+
+import static java.lang.Integer.parseInt; // violation
+
 /**
 * @link List
```

Comment:
yes, this is invalid but the check counted this as usage I just noticed that It should be ignored according to Import Check but it isn't
this could be a new issue or just update this example to be a valid link?

Reply:
yes, this is invalid but the check counted this as usage I just noticed that I think It should be ignored according to Import Check but it isn't
this could be a new issue or just update this example to be a valid link?

Reply:
@rnveach I update it to be a valid link in the docs examples but I think this should be an issue in the check.is it normal to consider this as usage?

Reply:
Please make a new issue for fixing the check. The check itself shouldn't be fixed in this issue.

Reply:
@rnveach Yes I merged them as one issue because I think that both examples are related. the issue is that the check not detect an invalid link if the tagName is correct and this is the case in both examples